poll/translations/en/LC_MESSAGES/text.po

```diff
@@ -0,0 +1,231 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: \n"
+"POT-Creation-Date: 2016-04-07 16:17+0500\n"
+"PO-Revision-Date: 2016-04-07 16:17+0500\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"Language: en\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 1.8.7\n"
+"X-Poedit-Basepath: ../../..\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"X-Poedit-SearchPath-0: poll.py\n"
+
+#: poll.py
+msgid "Whether or not to display results to the user."
+msgstr ""
+
+#: poll.py
+msgid "The maximum number of times a user may send a submission."
+msgstr ""
+
+#: poll.py
+msgid "Number of times the user has sent a submission."
+msgstr ""
+
+#: poll.py
+msgid "Text to display after the user votes."
+msgstr ""
+
+#: poll.py
+#, python-brace-format
+msgid "'{0}' is not present, or not a JSON array."
+msgstr ""
+
+#: poll.py
+#, python-brace-format
+# translators: {noun} is either "Answer" or "Question"
```

Comment:
@asadiqbal08 AFAIK translator comments should be placed in source files, with the string definitions that they belong to. It's not enough to change the placeholders here; they need to be changed in the code as well. (In fact, you only need to change them in the code; you should get correct results for the translation files when you regenerate them afterwards.) Also, this particular string contains two named placeholders -- it would be worth explaining both of them. Example:

```python
                    # Translators: {noun} is either "Answer" or "Question". {item} identifies the answer or question.
                    self.ugettext("{noun} {item} not a javascript object!").format(noun=noun, item=item))
```

Since you've already gotten started on this, can you make the necessary changes for the remaining strings that currently don't use named placeholders? I had a look at polls.py, there's only ~5 more strings that you'd need to update. It would be great not to have to create a separate ticket for this (or remember to take care of it the next time we work on translating this XBlock).